### DISCUSSION TITLE
Missing Trash bin icon

### DISCUSSION THREAD

**Discussion by hencomb** [lawand-dot-io/taskbar#69] [Feature requests]
**Describe the bug**
Default installation is missing the Trash bin icon and nowhere in preferences to switch it on.

**To Reproduce**
Install

**Expected behavior**
Trash bin icon should be visible and usable.

**Screenshots**

**Desktop (please complete the following information):**
 - Taskbar version 1.2.1
 - OS: Sonoma 14.5
 - Chip Intel i7

**Additional context**

**Comment by hencomb** [comment]
ok, found it. It's in the Menu.

**Comment by lawand-dot-io** [comment]
@hencomb what did you find? I don't remember a menu option for a trash icon

**Comment by hencomb** [comment]
You can ignore this ticket. I closed it once I found how the taskbar shows
Trash

Thanks

**Comment by lawand-dot-io** [comment]
I don't think there's a way to show Trash on Taskbar, that's why I'm
confused

**Comment by lawand-dot-io** [comment]
This is a duplicate of: https://github.com/lawand-dot-io/taskbar/discussions/12
